MSFragger-GUI/src/com/dmtavt/fragpipe/FragpipeRun.java

```diff
@@ -1422,7 +1422,8 @@ private static boolean configureTaskGraph(JComponent parent, Path wd, Path jarPa
             msBoosterPanel.findBestImModel(),
             msBoosterPanel.koinaUrl(),
             msBoosterPanel.rtTestModels(),
-            msBoosterPanel.spectraTestModels());
+            msBoosterPanel.spectraTestModels(),
+            msBoosterPanel.imTestModels());

```

Comment:
Is it used anywhere?

Reply:
Yes, DIA-NN or AlphaPeptDeep can both predict IM/CCS

Reply:
I am not sure if there is any miscommunication. It seems that MSBooster does not have a `imSearchModelsString` parameter similar to spectral intensity and RT. Your commits also didn't add this option. Thus, the `msBoosterPanel.imTestModels()` is never used.

Reply:
I think it is being set properly when I run it

Reply:
OK, you added the option in https://github.com/Nesvilab/FragPipe/pull/1805/commits/bbbbe572d7fafd9c56982dc887fb829dfa2b76bf and https://github.com/Nesvilab/FragPipe/pull/1805/commits/2964fd39e47c317c8a88e7cc08d0d35bab326962. It was not there before and you never told me that you have this option.